addons/note/static/src/js/systray_activity_menu.js

```diff
@@ -54,15 +54,7 @@ ActivityMenu.include({
     //-----------------------------------------
     // Handlers
     //-----------------------------------------
-    /**
-     * @override
-     */
-    _onActivityFilterClick(ev) {
-        const $el = $(ev.currentTarget);
-        if (!$el.hasClass("o_note")) {
-            this._super.apply(this, arguments);
-        }
-    },
```

Comment:
I think you still need to patch in `note` so that `o_note` click filter does nothing, no?

Reply:
Events used to be bound based on selectors, they are now bound on template elements. Note was sharing the class (still does, I did minimal diff in the module), but not the events.